README.md

```diff
@@ -313,19 +313,13 @@ Finally: your plugin's `package.json` must expose each UI extension as a sub-mod
     // Main module entry point – the default Astro Integration export
     ".": "./dist/index.js",
     // Module export for the HelloWorldMessage UI extension
-    "./HelloWorldMessage": "./dist/extensions/HelloWorldMessage.js"
+    "./HelloWorldMessage": "./dist/HelloWorldMessage.js"
   }
 }
 ```
 
 Congratulations. This sets up the foundation for your plugin!
 
-The build step requires [copyfiles](https://www.npmjs.com/package/copyfiles) if you do not have that installed run the following:
```

Comment:
Good point... `copyfiles` is a package I happened to have installed globally, so this step was probably for you. 

Rather than installing it globally, it's probably better to just add it as a dev dependency. I've done that above, and removed this section.